lms/views/lti/deeplinking.py

```diff
@@ -2,42 +2,51 @@
 Views for handling content item selection / deep linking launches.
 
 A content item selection request is an LTI launch request (so it's a form
-submission containing all the usual launch params, including the OAuth 1
-signature) that's used by LMS's during the assignment creation process.
+submission containing all the usual launch params) that's used
+by LMS's during the assignment creation process.
 
 When an instructor creates a Hypothesis assignment in an LMS that supports
 content item selection the LMS launches our app in order for us to present an
 interface for the user to select a "content item" (in our case: a document to
 be annotated) for use with the assignment.
 
-The spec requires that content item selection requests have an
-``lti_message_type`` parameter with the value ``ContentItemSelectionRequest``,
+The spec requires that content item selection launches have an
+``lti_message_type`` indicating that this is indeed  a content-item request
 but we don't actually require the requests to have this parameter: instead we
 use a separate URL to distinguish content item selection launches.
 
 When the user selects a document we get the browser to POST the selection back
 to the LMS in a form submission with the ``lti_message_type`` parameter set to
-``ContentItemSelection``. The original ``ContentItemSelectionRequest``
-launch's ``content_item_return_url`` parameter gives us the URL to POST this
+``ContentItemSelection`` (LTI1.1) or ``LtiDeepLinkingRequest`` (LTI1.3).
+
+The launch's ``content_item_return_url`` parameter gives us the URL to POST this
 form submission to. The LMS saves the selection and passes it back to us in the
 launch params whenever this assignment is launched in future.
 
-For more details see the LTI Deep Linking spec:
+For more details see the LTI Deep Linking specs:
+
+- LTI 1.1 https://www.imsglobal.org/specs/lticiv1p0
+
+  Especially this page:
 
-https://www.imsglobal.org/specs/lticiv1p0
+  https://www.imsglobal.org/specs/lticiv1p0/specification-3
 
-Especially this page:
+- LTI 1.3 https://www.imsglobal.org/spec/lti-dl/v2p0
 
-https://www.imsglobal.org/specs/lticiv1p0/specification-3
 
 Canvas LMS's Content Item docs are also useful:
 
 https://canvas.instructure.com/doc/api/file.content_item.html
```

Comment:
Used a combination or yours and the existing one :+1:

Reply:
> * In the case of LTI1.1:
    * ??? TBA - I'm not really sure what happens here?

There's another follow up PR (mentioned on the PR description but it needs heavy rebasing) to follow the same approach in LTI1.1.

In the backend it mostly adding the v11 view which instead of a JWT returns a few json fields. Currently the frontend constructs that message itself.

On the frontend it will involve mostly removing code but I wanted to have this PR first setting the machinery in both the backend/frontend (and having low risk as is LTI1.3 only) and in the follow up just adapt the existing code and focus more on testing the LTI1.1 changes.